fog/block_provider/src/local.rs

```diff
@@ -14,8 +14,8 @@ use std::time::Duration;
 
 #[derive(Clone)]
 pub struct LocalBlockProvider<L: Ledger + Clone + Sync> {
-    ledger: L,
-    watcher: Option<WatcherDB>,
+    pub ledger: L,
+    pub watcher: Option<WatcherDB>,
```

Comment:
The LocalBlockProvider is pretty much only used in tests now.
I needed mutable access to `ledger` so instead of creating a mutable accessor method chose to make the field public.